acceptance/helpers.rb

```diff
@@ -23,8 +23,8 @@ def collection_cmp(col_1, col_2)
 
         return 0 if col_1 == col_2
 
-        return -1 if col_1 == 'pc1'
-        return 1 if col_2 == 'pc1'
+        return -1 if col_1 =~ /pc1/i
+        return 1 if col_2 =~ /pc1/i
```

Comment:
Are these changes covered by tests?

And is the only intended change here to handle mixed case? `casecmp` might be a better candidate if that's so.

Reply:
Well, there are no tests for these helpers, but this method is used in the acceptance tests to restrict which tests run based on the version of puppetserver installed on the master -- that functionality still works for me. This change is intended to allow for using either PC1 (the "official" name of the collection) or pc1 (required in some download URLs, so also common).

Reply:
oh, I totally igonred the second part of your comment -- I'll use casecmmp